Kitodo/src/main/java/org/goobi/mq/processors/CreateNewProcessProcessor.java

```diff
@@ -78,8 +77,8 @@
  * 			<dd>Fields to be populated manually.
  * 		</dd>
  * </dl>
- * 
- * @author Matthias Ronge <matthias.ronge@zeutschel.de>
+ *
+ * @author Matthias Ronge 	&lt;matthias.ronge@zeutschel.de&gt;
```

Comment:
There are two reasons to use html entities.
1. Checkstyle treats it as a errors <error line="5" severity="warning" message="Javadoc comment at column 58 has parse error. Details: no viable alternative at input &apos;&lt;contact@&apos; while parsing HTML_ELEMENT" source="com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck"/>
2. It is necessary for correct generation of JavaDoc by generators.